settings.gradle.kts

```diff
@@ -40,6 +40,7 @@ rootProject.name = "valkyrie"
 include("idea-plugin")
 
 include("components:androidx-compose")
+include("components:extensions")
```

Comment:
If this is designed for tests, could be renamed to `components:test-utils`

Reply:
this will include other reused extensions across modules